content/en/_index.html

```diff
@@ -173,12 +171,12 @@ <h5 class="card-title text-white section-head">Community</h5>
   <div class="contain">
     <h4 class="section-head">Join our community</h4>
     <p>
-      Want to contribute to Kubeflow? <a href="/docs/about/community/">Join our community</a>! Check out upcoming meetings, get involved in discussions on our mailing list or on Slack, and suggest exciting new integrations.
+      Check out the <a href="/docs/about/kubeflow-community-call/">weekly community call</a>, get involved in discussions on the <a href="/docs/about/kubeflow-mailing-list/">mailing list</a> or chat with others on the <a href="/docs/about/kubeflow-slack-workspace/">Slack Workspace</a>!
```

Comment:
As you might have already noticed, we have many pages in kubeflow.org, it will be awesome to make relevant information more grouped, than having to navigate among many pages like this sentence tries to do.

Reply:
See my thoughts here: https://github.com/kubeflow/website/pull/3007#issuecomment-943762457

Reply:
I have changed it back to a single community page.